### PR TITLE
Creating models and migrations

### PR DESCRIPTION
**Pull request by bendeloyola**
Generating models and migration

rails g model Portfolio 
rails g model Transaction
rails g model Stocks

### CODE REVIEW

**Review by neroneronomi — APPROVED** [review]

File: db/migrate/20210911024734_create_portfolios.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreatePortfolios < ActiveRecord::Migration[6.0]
+  def change
+    create_table :portfolios do |t|
+      t.decimal :price
+      t.integer :qty
+
+      t.timestamps
+    end
+  end
+end
```
Comment (analyn-cajocson) [inline db/migrate/20210911024734_create_portfolios.rb:10]: I think you're missing some important FK here

File: db/migrate/20210911025137_create_transactions.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateTransactions < ActiveRecord::Migration[6.0]
+  def change
+    create_table :transactions do |t|
+      t.decimal :price
+      t.string :status
+
+      t.timestamps
+    end
+  end
+end
```
Comment (analyn-cajocson) [inline db/migrate/20210911025137_create_transactions.rb:10]: You're missing some important FK here

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: db/migrate/20210911024734_create_portfolios.rb (hunk 1 above)
Comment (bendeloyola) [reply db/migrate/20210911024734_create_portfolios.rb:10]: added foreign key

File: db/migrate/20210911025137_create_transactions.rb (hunk 2 above)
Comment (bendeloyola) [reply db/migrate/20210911025137_create_transactions.rb:10]: Added foreign key

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**